repo

```diff
@@ -4,9 +4,9 @@ upstream: "https://github.com/ocaml/opam-repository/tree/master/"
 announce: [
 """
 [WARNING] opam is out-of-date. Please consider updating it (https://opam.ocaml.org/doc/Install.html)
```

Comment:
Apropos [this Discuss thread](https://discuss.ocaml.org/t/spurious-opam-is-out-of-date-message/15183), I think it would be sensible to include the “offending” opam version in the message:
```suggestion
[WARNING] opam %{opam-version}% is out-of-date. Please consider updating it (https://opam.ocaml.org/doc/Install.html)
```